client/context/context.go

```diff
@@ -7,6 +7,8 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/pkg/errors"
```

Comment:
do we use this package somewhere else too?

Reply:
AFAIK that package has entered _maintenance mode_ some time ago: https://github.com/pkg/errors#roadmap

Reply:
Yeah, it's a simple error wrapper. And yes, we use it elsewhere.

Reply:
```
alessio@bangalter:~/work/cosmos-sdk$ grep -rw 'github.com/pkg/errors' 
Binary file build/gaiad matches
Binary file build/gaiakeyutil matches
Binary file build/gaiacli matches
Binary file build/gaiareplay matches
Binary file cmd/gaia/build/gaiad matches
Binary file cmd/gaia/build/gaiakeyutil matches
Binary file cmd/gaia/build/gaiacli matches
Binary file cmd/gaia/build/gaiareplay matches
types/errors.go:	"github.com/pkg/errors"
types/dec_coin.go:	"github.com/pkg/errors"
client/context/query.go:	"github.com/pkg/errors"
client/context/context.go:	"github.com/pkg/errors"
go.sum:github.com/pkg/errors v0.8.0 h1:WdK/asTD0HN+q6hsWO3/vpuAkAr+tw6aNJNDFFf0+qw=
go.sum:github.com/pkg/errors v0.8.0/go.mod h1:bwawxfHBFNV+L2hUp1rHADufV3IMtnDRdf1r5NINEl0=
crypto/keys/keybase.go:	"github.com/pkg/errors"
crypto/ledger_secp256k1.go:	"github.com/pkg/errors"
crypto/ledger_mock.go:	"github.com/pkg/errors"
crypto/ledger_notavail.go:	"github.com/pkg/errors"
go.mod:	github.com/pkg/errors v0.8.0
```

It's quite unfortunate that we don't enforce it then as it seems useful.

Reply:
what's the difference with `fmt.Errorf()` ?

Reply:
Stack traces mostly. Also from Dave Chaney:

```
If you interact with a package from another repository, consider using errors.Wrap or errors.Wrapf to establish a stack trace at that point. This advice also applies when interacting with the standard library.
```